src/js/common/schemaform/fields/AutosuggestField.jsx

```diff
@@ -156,24 +148,46 @@ export default class AutosuggestField extends React.Component {
     }
 
     return (
-      <Autosuggest
-        getSuggestionValue={suggestion => suggestion.label}
-        highlightFirstSuggestion
-        onSuggestionsClearRequested={this.handleSuggestionsClearRequested}
-        onSuggestionSelected={this.handleSuggestionSelected}
-        onSuggestionsFetchRequested={this.handleSuggestionsFetchRequested}
-        renderSuggestion={this.renderSuggestion}
-        shouldRenderSuggestions={this.shouldRenderSuggestions}
-        suggestions={this.state.suggestions}
-        inputProps={{
-          id,
-          name: id,
-          value: this.state.input,
-          onChange: this.onChange,
-          onKeyDown: this.onKeyDown,
-          'aria-labelledby': `${id}-label`,
-          onBlur: this.handleBlur
-        }}/>
+      <Downshift
+        onChange={this.handleChange}
+        onInputValueChange={this.handleInputValueChange}
+        inputValue={this.state.input}
+        selectedItem={this.state.input}
+        itemToString={item => {
+          if (typeof item === 'string') {
+            return item;
+          }
+
+          return item.label;
+        }}
+        render={({
+          getInputProps,
+          getItemProps,
+          isOpen,
+          selectedItem,
+          highlightedIndex
+        }) => (
+          <div className="autosuggest-container">
+            <input {...getInputProps({ id, name: id, onKeyDown: this.handleKeyDown, onBlur: this.handleBlur })}/>
+            {isOpen && (
+              <div className="autosuggest-list">
```

Comment:
Running aXe coconut indicates that this div should be given the listbox role:
>Certain ARIA roles must contain particular children
Issue description
Ensures elements with an ARIA role that require child roles contain them

>Impact: critical
 Learn more
Element location
#root_application_veteran_serviceRecords_0_serviceBranch
Element source
<input role="combobox" aria-autocomplete="list" aria-expanded="false" autocomplete="off" value="army" id="root_application_veteran_serviceRecords_0_serviceBranch" name="root_application_veteran_serviceRecords_0_serviceBranch">

>To solve this violation, you need to:
Fix the following:
Required ARIA child role not present: listbox